digdag-cli/src/main/resources/digdag/cli/selfcopy.bat

```diff
@@ -0,0 +1,11 @@
+@echo off
+set i=0
+:repeat
+  if %i% leq 30 (goto move) else goto :done
+:move
+  ping 1.1.1.1 -n 1 -w 500 >NUL 2>NUL
+  move /Y %1 %2 >NUL 2>NUL
+  set /a "i = i + 1"
+  if exist %1 goto repeat
+:done
+del %0

```

Comment:
Putting `del %0` at the head didn't work.
What I found instead is that we should use `%~f0` instead of `%0`: http://stackoverflow.com/questions/2888976/how-to-make-bat-file-delete-it-self-after-completion